src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -101,7 +102,10 @@ object WorkflowDescriptor {
     validateWorkflowDescriptor(id, sourceFiles, CromwellBackend.backend(), conf) match {
       case scalaz.Success(w) => w
       case scalaz.Failure(f) =>
-        throw new IllegalArgumentException(s"""Workflow $id failed to process inputs:\n${f.toList.mkString("\n")}""")
+        throw new IllegalArgumentException() with ThrowableWithErrors {
+          val message = s"Workflow input processing failed."
```

Comment:
Might this ID be useful to ops?  i.e. have we possibly generated any other log messages with this ID before we got to this point?  Hopefully there's enough context left in the errors to provide a connection with the submission.